Rollbar/NetStandard/IRollbarConfigurationLoader.cs

```diff
@@ -0,0 +1,37 @@
+﻿namespace Rollbar.NetStandard
+{
+    using Rollbar.Telemetry;
+
+    /// <summary>
+    /// Interface IRollbarConfigurationLoader
+    /// </summary>
+    public interface IRollbarConfigurationLoader
+    {
+        /// <summary>
+        /// Loads the povided configuration object based on found configuration store (if any).
```

Comment:
Nit: typo, "povided" should be "provided" I think.

Reply:
corrected. thanks!